test/dump/global.txt

```diff
@@ -1,6 +1,11 @@
 ;;; TOOL: run-wasmdump
 ;;; FLAGS: -v --dump-verbose
 (module
+  (import "foo" "i32" (global i32))
```

Comment:
Maybe give these names that are not identical to the type symbols?  maybe i32_import?